spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { described_class.new(name: 'John') }
+
+  describe 'associations' do
+    it { should have_many(:posts).with_foreign_key(:author_id).dependent(:destroy) }
+    it { should have_many(:comments).with_foreign_key(:author_id) }
+    it { should have_many(:likes).with_foreign_key(:author_id).class_name('Like').dependent(:destroy) }
```

Comment:
- **[OPTIONAL]** This test returns an error, please fix it to pass them all.

![image](https://github.com/Zeph180/blog-it/assets/58642949/1bbe1f6f-f8a6-4d0f-a729-8119dfc5a785)